.mergify.yml

```diff
@@ -2,8 +2,8 @@
 pull_request_rules:
   - name: Merge PRs that are ready
     conditions:
-      - status-success=Travis CI - Pull Request
-      - status-success=typesafe-cla-validator
+      - check-success~=/ Ready To Merge$
+      - check-success=typesafe-cla-validator
```

Comment:
This is not the only thing that needs to be changed in the mergify conf.
You can actually just copy the conf from twirl: https://github.com/playframework/twirl/blob/main/.mergify.yml
As you can see it adds `queue_rules` add the top and then uses this queue in `actions`. The `merge` action is deprecated AFAIK.